common/common.go

```diff
@@ -50,3 +50,9 @@ func IsGHError(err error) bool {
 	_, ok := err.(*github.ErrorResponse)
 	return ok
 }
+
+// IsGHRateLimitError checks if the error is a github rate limit error
+func IsGHRatelimitError(err error) bool {
```

Comment:
I don't think we need this as we're handling it when there is any error from github